outputs/mainnet.json

```diff
@@ -1594,13 +1595,135 @@
+  "pools/WETH-EIGEN-b57c9": "0xb57c9284C615E4d3E3A26f418ab32836DCe0133f",
+  "pools/WETH-EIGEN-b5a8e": "0xb5a8E9f72D735B18aC55F0E6b56653Dfae599A4F",
+  "pools/WETH-EIGEN-b6c73": "0xB6C7366457D796a8bC572D22Afa48886aA5AAa47",
+  "pools/WETH-EIGEN-b8288": "0xb8288872Fa40f6350DcB8C9A66c8Dd439e707f76",
+  "pools/WETH-EIGEN-b8a26": "0xb8A261f3680ACDCf1056F9ee8bf372fa0b5d6206",
+  "pools/WETH-EIGEN-b8cbe": "0xB8cbe6004a00F55dDF34682F6fe45dE68a113F63",
+  "pools/WETH-EIGEN-b96da": "0xB96Dac9051653eBf20C081d1F902170c11B4589A",
+  "pools/WETH-EIGEN-b9e3a": "0xb9e3Ad4D3035FB9801079073D8D9745eb2E0a035",
+  "pools/WETH-EIGEN-ba0f1": "0xBA0F118fad469bA04d5e997d474304B5e67fe5b0",
+  "pools/WETH-EIGEN-bab32": "0xbAb3292F9dd3d3aD035288a1D5Be92057D6B5d2B",
+  "pools/WETH-EIGEN-bf5e2": "0xBF5E285B45Fb25e46659D660a3438C2632895F1E",
+  "pools/WETH-EIGEN-c9718": "0xC97182ddB330a165eaF322651930DbA0a8a40E48",
+  "pools/WETH-EIGEN-c971e": "0xC971ecF12c7A201a26e3d46a612Add9CB1D5b086",
+  "pools/WETH-EIGEN-cf2c3": "0xCf2c39A0c9097fC6fD3C813CaFAAB0dcf0604Bf5",
+  "pools/WETH-EIGEN-d2fd7": "0xd2FD71442297FB0541D22274572d1f440C545994",
+  "pools/WETH-EIGEN-d3508": "0xD350814bDA540E05A78A86fBc6bB4448783cA949",
+  "pools/WETH-EIGEN-d5249": "0xd5249B5cEBAaa4409a343F9b519B97491e9d9cC0",
+  "pools/WETH-EIGEN-d6a99": "0xd6a9921eE972ff7379D35ac8E4EC1D5C71893Bd4",
+  "pools/WETH-EIGEN-e0940": "0xe09406AF75Cd5de660aBf11AF0FD87c861851884",
+  "pools/WETH-EIGEN-e250b": "0xE250B38De9f6F3B8CC78fed6B06B51E320a734AC",
+  "pools/WETH-EIGEN-e2df7": "0xe2DF7dF1aE894Ac17eA4E6251BCe1432B875df34",
+  "pools/WETH-EIGEN-e5bc2": "0xE5bC246c3cFb130BCc1B67FF397a6BD1a2e9cEA6",
+  "pools/WETH-EIGEN-e69d2": "0xE69D23b4B70AFA1153BA68F77ad3CaaA955C1c3f",
+  "pools/WETH-EIGEN-e753b": "0xe753B7b708feb59215e0161D4628a9594E5EBFc3",
+  "pools/WETH-EIGEN-e89a6": "0xE89a636d6Ba000048054185806ff01e72D8e0b7c",
+  "pools/WETH-EIGEN-eb0a9": "0xeb0a9432794cC0c8170f73359E4F4255c991820d",
+  "pools/WETH-EIGEN-ed0cb": "0xED0Cb532A682694f6926da2b7Edf248Ed482dc94",
+  "pools/WETH-EIGEN-ed2b4": "0xeD2B4F786ac2Dd4a775294c47866C8eF9b3A094B",
+  "pools/WETH-EIGEN-ed574": "0xED574006190F0DcAF1743CD968F785d11E002761",
+  "pools/WETH-EIGEN-eeef1": "0xeeeF18bb05F8f819A97448e762E2bAbaeAE3D918",
+  "pools/WETH-EIGEN-ef7f2": "0xEF7F2825e0e106535d4086C0d5B9842fEf1E71A3",
+  "pools/WETH-EIGEN-f0494": "0xF0494212E114e09844734014Ec6c83ed29a0c04f",
+  "pools/WETH-EIGEN-f1c1a": "0xf1C1aA69872D546A037c5f24161732bb368Da6f2",
+  "pools/WETH-EIGEN-f294b": "0xF294B1DC61b16668749d7764A77119153f596EB5",
+  "pools/WETH-EIGEN-f2e8b": "0xF2e8bCA4E098D76625da442AA6d7A910dB6C8046",
+  "pools/WETH-EIGEN-f543e": "0xF543e5853616d3D6e69df75b96d1c299D55DA7Ba",
+  "pools/WETH-EIGEN-fb2a5": "0xFb2A50433A2ab2EB67Fb923aD5C2B47A1a2a0734",
+  "pools/WETH-EIGEN-fb4af": "0xfb4aFfbDF4b1b0100a9Cd27F2d742514475eA55e",
+  "pools/WETH-EIGEN-fb7ca": "0xfB7cAf2EB30EA80b1D109D64f83Fd72bd08caa1C",
+  "pools/WETH-EIGEN-fe0f5": "0xFe0F59cDdE96075E1F8216d8462C5EDa3ab80d2E",
```

Comment:
seems like the real issue is a great amount of WETH-EIGEN pools?

should we just filter those out instead, similar to what we did for the circle pools (https://github.com/BalancerMaxis/bal_addresses/pull/678)?